aries-core/src/js/components/core/Header/Header.js

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Header, ResponsiveContext } from 'grommet';
+
+const PAD_SIZES = ['xxsmall', 'xsmall', 'small', 'medium', 'large', 'xlarge'];
+
+const StyledHeader = ({ background, pad, ...rest }) => {
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <Header
+      background={background}
+      pad={
+        pad || {
+          vertical: 'medium',
+          horizontal: size !== 'small' ? 'xlarge' : 'large',
+        }
+      }
```

Comment:
I'm not sure the `size` adjustment assumption is right for the core library, different users may want to control the responsiveness of their app in different ways.

Reply:
Users can supply pad on their own if they want to override, these would be the default paddings. I think we want to have the design system to define this padding, but there is still option provided to override if wanted.

Reply:
I don't think that a whole component should exist for the sake of padding during responsive behavior, but I might be wrong here, would like to hear more thoughts.

Reply:
I think that's a valid point, padding for a general header could probably be taken care of in theming. I think we should discuss what you brought up previously (and what Matt mentioned below) about maybe this component in the way it's being used is something more specific and would be "Header" under the hood but named something else, maybe it requires the "AppIdentity" child or has other specifications.